src/ScheduleFunctions.cpp

```diff
@@ -72,17 +89,16 @@ bool contains_impure_call(const Expr &expr) {
     return is_not_pure.result;
 }
 
-// Build a loop nest about a provide node using a schedule
-Stmt build_provide_loop_nest_helper(string func_name,
-                                    string prefix,
-                                    int start_fuse, // Fuse the dims starting from start_fuse to outermost (if not negative)
-                                    const vector<string> &dims, // The pure dims
-                                    const vector<Expr> &site,
-                                    const vector<Expr> &values,
-                                    const vector<Expr> &predicates,
-                                    const FuncSchedule &func_s,
-                                    const StageSchedule &stage_s,
-                                    bool is_update) {
+/**
+ *  Build a loop nest about a provide node using a schedule
+ * @param start_fuse The index of the dimension in \p dims, or the length of \p dims if not fusing
+ * @param dims The pure dimensions
+ * @return The loop nest containing the provide node for the given func/stage
+ */
+Stmt build_provide_loop_nest_helper(const string &func_name, const string &prefix, int start_fuse,
```

Comment:
As discussed elsewhere, would prefer one arg per line whenever args are wrapped (not a hard rule - there may be some natural semantically-meaningful grouping of the args).